### PR TITLE
build: upgraded the plug library to 1.14.0 to remove a warning

### PR DESCRIPTION
**Pull request by etorreborre**

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
wonder why dependabot didn't propose this upgrade 🤔 @metaclips

**Comment by etorreborre** [comment]
@mrinalwadhwa this is still safe to merge, isn't it?

**Comment by mrinalwadhwa** [comment]
Yes

File: implementations/elixir/ockam/ockam_cloud_node/mix.lock [hunk 1]
```diff
@@ -36,7 +36,7 @@
...
   "parse_trans": {:hex, :parse_trans, "3.3.1", "16328ab840cc09919bd10dab29e431da3af9e9e7e7e6f0089dd5a2d2820011d8", [:rebar3], [], "hexpm", "07cd9577885f56362d414e8c4c4e6bdf10d43a8767abb92d24cbe8b24c54888b"},
-  "plug": {:hex, :plug, "1.13.6", "187beb6b67c6cec50503e940f0434ea4692b19384d47e5fdfd701e93cadb4cc2", [:mix], [{:mime, "~> 1.0 or ~> 2.0", [hex: :mime, repo: "hexpm", optional: false]}, {:plug_crypto, "~> 1.1.1 or ~> 1.2", [hex: :plug_crypto, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.4.3 or ~> 1.0", [hex: :telemetry, repo: "hexpm", optional: false]}], "hexpm", "02b9c6b9955bce92c829f31d6284bf53c591ca63c4fb9ff81dfd0418667a34ff"},
+  "plug": {:hex, :plug, "1.14.0", "ba4f558468f69cbd9f6b356d25443d0b796fbdc887e03fa89001384a9cac638f", [:mix], [{:mime, "~> 1.0 or ~> 2.0", [hex: :mime, repo: "hexpm", optional: false]}, {:plug_crypto, "~> 1.1.1 or ~> 1.2", [hex: :plug_crypto, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.4.3 or ~> 1.0", [hex: :telemetry, repo: "hexpm", optional: false]}], "hexpm", "02b9c6b9955bce92c829f31d6284bf53c591ca63c4fb9ff81dfd0418667a34ff"},
```
Comment (mrinalwadhwa) [inline implementations/elixir/ockam/ockam_cloud_node/mix.lock:39]: checked https://hex.pm/packages/plug/1.14.0

**Review by mrinalwadhwa — DISMISSED** [review]

**Comment by metaclips** [comment]
> wonder why dependabot didn't propose this upgrade 🤔 @metaclips

I'll take a look

**Review by mrinalwadhwa — DISMISSED** [review]

File: implementations/elixir/ockam/ockam_cloud_node/mix.lock (hunk 1 above)
Comment (etorreborre) [reply implementations/elixir/ockam/ockam_cloud_node/mix.lock:39]: Thanks @mrinalwadhwa I don't know how I missed that one.

**Review by metaclips — APPROVED** [review]

**Review by mrinalwadhwa — APPROVED** [review]

**Pull request merged**